CAP/gap/CategoryMorphisms.gi

```diff
@@ -566,3 +566,14 @@ InstallMethod( String,
     return Concatenation( "A morphism in ", Name( CapCategory( morphism ) ) );
     
 end );
+
+InstallMethod( Display,
+               [ IsCapCategoryMorphism and HasCapCategory ],
```

Comment:
```suggestion
               [ IsCapCategoryMorphism ],
```

I suggest to delete the filter `HasCapCategory` here.
- Advantage: in instances of categories for which we provide the `Display` method, this generic `Display` will not be judged by GAP as being "more specific"
- Disadvantage: this method will throw an error if you input a morphism that doesn't have a Cap category.

In my opinion, the disadvantage is not too bad since the error message will be: no method found for `CapCategory`, which exactly points to the semantical error someone made in such a situation:
every GAP object in the filter `IsCapCategoryMorphism` is supposed to have a `CapCategory`.